packages/api-specification/interface/window-extension.ts

```diff
@@ -0,0 +1,8 @@
+declare interface Window {
```

Comment:
Do we still need this window-extension? I thought it related to the old way of opening a notification (and we now have ssf.Notification instead)... unless there's another need for it?